src/Workspaces/CoreTest/SolutionTests/SolutionTests.cs

```diff
@@ -1497,7 +1502,7 @@ public void TestWithSyntaxTree()
             var factory = dummyProject.LanguageServices.SyntaxTreeFactory;
 
             // create the origin tree
-            var strongTree = factory.ParseSyntaxTree("dummy", dummyProject.ParseOptions, SourceText.From("// emtpy"), analyzerConfigOptionsResult: null, CancellationToken.None);
+            var strongTree = factory.ParseSyntaxTree("dummy", dummyProject.ParseOptions, SourceText.From("// emtpy"), analyzerConfigOptionsResult: null, isGenerated: null, CancellationToken.None);
```

Comment:
Revert this change? This seems to be causing build failure in the PR.

Reply:
Ah, yes. Thanks!